### ISSUE TITLE
Possible regression in inspectdb with schema argument

### ISSUE THREAD

**Issue by Wedge009** [microsoft/mssql-django#411]
**Software versions**
* Django: 5.0.8
* mssql-django: 1.5
* python: 3.12.5
* SQL Server: 2016 (13.0.7037)
* OS: Windows 11

**Table schema and Model**
N/A

**Database Connection Settings**
Blocked by stupid corporate proxy, but there's nothing relevant here, I believe.

**Problem description and steps to reproduce**
`python manage.py inspectdb` works as expected - demonstrating the database connection is valid - but only for the default schema.

`python manage.py inspectdb --schema [schema_name]` results in the following error:

```
usage: manage.py inspectdb [-h] [--database DATABASE] [--include-partitions] [--include-views] [--version] [-v {0,1,2,3}] [--settings SETTINGS] [--pythonpath PYTHONPATH] [--traceback]
                           [--no-color] [--force-color]
                           [table ...]
manage.py inspectdb: error: unrecognized arguments: --schema
```

**Expected behaviour and actual behaviour**
* Database inspection with specified schema.
* Aforementioned error.

**Error message/stack trace**
As above.

**Any other details that can be helpful**
Formal report of the problem that I mentioned previously at https://github.com/microsoft/mssql-django/issues/56#issuecomment-2306027381

**Comment by mShan0** [comment]
Thanks for the report. Will look into this

**Comment by Wedge009** [comment]
Thank you for the acknowledgement.

**Comment by Wedge009** [comment]
Noting that this appears to be still an issue with mssql-django 1.6 on Django 5.2.5.

**Comment by Wedge009** [comment]
Didn't notice that this is effectively a duplicate of #183. As noted there, `'mssql'` needs to be included in Django's installed applications.
